html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello About Me!</p>
+  <img src="https://www.mlive.com/resizer/HmVkldbVbQ08gk0FCJSZo-ZRVfY=/1280x0/smart/cloudfront-us-east-1.images.arcpublishing.com/advancelocal/HB6R5CPZ7RCGXN6MAHIAHLYHVY.jpg" alt="Detroit Red Wings Hockey Team">
```

Comment:
Please use a relative url to the images provided in the starter files.

Reply:
<img width="707" alt="Screen Shot 2022-01-25 at 11 13 59 AM" src="https://user-images.githubusercontent.com/96929704/151043807-d1599c1a-1f64-4017-b48a-82213f1c165c.png">